app/config/FrontendAppConfig.scala

```diff
@@ -48,6 +49,32 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
     "cy" -> Lang("cy")
   )
 
+  protected lazy val rootServices = "microservice.services"
+
+  protected lazy val defaultProtocol: String =
+    configuration
+      .getOptional[String](s"$rootServices.protocol")
+      .getOrElse("http")
+
+  def getConfString(confKey: String, defString: => String) =
+    configuration
+      .getOptional[String](s"$rootServices.$confKey")
+      .getOrElse(defString)
+
+  def getConfInt(confKey: String, defInt: => Int) =
+    configuration
+      .getOptional[Int](s"$rootServices.$confKey")
+      .getOrElse(defInt)
+
+  private def throwConfigNotFoundError(key: String) =
+    throw new RuntimeException(s"Could not find config key '$key'")
+  def baseUrl(serviceName: String) = {
+    val protocol = getConfString(s"$serviceName.protocol", defaultProtocol)
+    val host = getConfString(s"$serviceName.host", throwConfigNotFoundError(s"$serviceName.host"))
```

Comment:
can't we use ServicesConfig like below instead of all this manual things ? 

val pertaxUrl: String = s"${servicesConfig.baseUrl(serviceName = "pertax-frontend")}/personal-account"